test/exclude-members.test.js

```diff
@@ -606,3 +606,88 @@ test('Can exclude a person in a group with a lot of members', async (t) => {
 
   await Promise.all(all.map((peer) => p(peer.close)(true)))
 })
+
+test("restarting the client doesn't make us rejoin old stuff", async (t) => {
+  const alice = Testbot({
+    keys: ssbKeys.generate(null, 'alice'),
+    mfSeed: Buffer.from(
+      '000000000000000000000000000000000000000000000000000000000000a1ce',
+      'hex'
+    ),
+  })
+  let bob = Testbot({
+    name: 'bobrestart',
+    keys: ssbKeys.generate(null, 'bob'),
+    mfSeed: Buffer.from(
+      '0000000000000000000000000000000000000000000000000000000000000b0b',
+      'hex'
+    ),
```

Comment:
I think extract these as `bobConfig` and then re-use it below (with `{ rimraf: false, ...bobConfig }`